src/commands/notify.yml

```diff
@@ -85,6 +103,12 @@ steps:
           echo "Please input your SLACK_WEBHOOK value either in the settings for this project, or as a parameter for this orb."
           exit 1
         else
+          if [[ -n "<< parameters.when >>" && << parameters.when >> != "always" ]]; then
```

Comment:
I believe this step will need to be modified so this step itself possesses the "when:always" parameter. If the job has failed before the notification this step will not execute, only the setting failure condition step would have the chance to run.

Reply:
@cayter ^